img/f/freedommobile.ca.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 100 36"><defs><path id="a" d="M.323.327h16.151V22.42H.323z"/></defs><g fill="none" fill-rule="evenodd"><g transform="translate(60 .058)"><mask id="b" fill="#fff"><use xlink:href="#a"/></mask><path fill="#2F7DC0" d="M2.196.327 16.474 22.42S5.02 20.877 1.683 14.627c-3.336-6.25.513-14.3.513-14.3" mask="url(#b)"/></g><path fill="#2F7DC0" d="m73.087 1.18 3.387 21.297s-8.079-4.951-8.51-10.744c-.43-5.791 5.123-10.552 5.123-10.552"/><path fill="#00558C" d="M67.964 11.734c.431 5.792 8.51 10.743 8.51 10.743l-8.39-12.982a9.06 9.06 0 0 0-.12 2.239"/></g></svg>
```

Comment:
I think you tried using parts of an SVG without changing the viewBox (width/height) of the SVG. Here's one with with a changed viewBox:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16.15 22.09"><path fill="#2f7dc0" d="M1.87 0s-3.85 8.05-.5 14.3c3.33 6.25 14.78 7.8 14.78 7.8L12.77.8s-4.18 3.58-5 8.31z"/><path fill="#00558c" d="M7.64 11.35c.43 5.8 8.51 10.74 8.51 10.74L7.76 9.11a9.06 9.06 0 0 0-.12 2.24"/></svg>
```